mkdocs.yml

```diff
@@ -62,6 +62,11 @@ theme:
     repo: fontawesome/brands/git-alt
   name: material
   palette:
+    # Palette toggle for automatic mode
+    - media: "(prefers-color-scheme)"
+      toggle:
+        icon: material/brightness-auto
+        name: Switch to light mode
```

Comment:
Should this be "Switch to system preference" instead of "Switch to light mode"

Reply:
No. The label shows the next option that will be enabled if clicked.

I initially thought it was an error (taken from mkdocs documentation), "fixed" it, and then realized I introduced an issue.